docs/jumpstart/week-5.md

```diff
@@ -0,0 +1,582 @@
+to make a new function because `printHi()` only prints "hi". We would then have to make `printHey()`.
+
+```
+function printHey() {
+
+    console.log("hey");
+
+}
+```
+
+We have now printed "hi" and "hey," but it is very repetitive and not efficient. If we wanted to print another
+message, we would have to make a new function OR we can just add an argument to a function to print whatever we
+want to the console and we would not need to make a new function each time.
+
+```
+function printMessage(message) {
+
+    console.log("Here is your note: " + message);
+
+}
+```
+
+Now we can reuse the function whenever we want to print something to the console! 
+
+### Using Props
+
+This same logic can be applied when we use props. Let us revisit the `LuckyNum` component again.
+
+```
+import React from 'react'; 
+
+const LuckyNum = () => {
+
+    let myLuckyNumber = 7;
+
+    return (
+        <div>
+            <h1>My Lucky Number is {myLuckyNumber}</h1>
+        </div>
+    );
+}
+
+export default LuckyNum;
+```
+
+Right now this component only displays John's lucky number. In order to make this component more reusable so that
+Doug can use it, we must add props. We add props as a argument of the function component.
+
+```
+import React from 'react'; 
+
+const LuckyNum = (props) => {
+
+    let myLuckyNumber = 7;
+
+    return (
+        <div>
+            <h1>My Lucky Number is {myLuckyNumber}</h1>
+        </div>
+    );
+}
+
+export default LuckyNum;
+```
+
+`props` is an object that contains every prop passed to the component. We can access a specific prop's value
+ by it's argument name. In our case, we can access the `luckyNum` field of the prop object and display that.
+ Now that we have access to `props` we can simply display the luckyNum field of the prop object.
+
+```
+import React from 'react'; 
+
+const LuckyNum = (props) => (
+
+    <div>
+        <h1>My Lucky Number is {props.luckyNumber}</h1>
+    </div>
+
+)
+
+export default LuckyNum;
+```
+Our component is now reusable!! We now can use it on Doug's profile like so:
+
+```
+import React from 'react';
+import LuckyNum from './LuckyNum';
+
+const Profile = () => (
+
+    <div>
+        <img src="profileImage.jpg"/>
+        <h1>Hi, my name is Doug! My favorite color is purple.</h1>
+        <LuckyNum luckyNum={1111}/>
+    </div>
+);
+
+export default Profile;
+```
+
+`props` is an object that React makes for each component. When you pass parameters into component declaration 
+like we did with `LuckyNum` React adds the parameter as a field of the object `props`. In order to get the field
+of the object we simply access the object's field like we would any other object. 
+
+> We can make the `Profile` component more reusable with props. Try now to add props to make `Profile` more
+>generic (possible answer below). 
+
+```
+import React from 'react';
+import LuckyNum from './LuckyNum';
+
+const Profile = (props) => (
+
+    <div>
+        <img src=`${props.profileURL}`/>
```

Comment:
Why are we using an interpolated string here rather than just doing `src={some_stuff}`?